Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocEnumValueControlObjectTest.cs

```diff
@@ -461,6 +461,9 @@ public void TestSelectOption ()
     [Test]
     public void TestSelectOptionPostBack ()
     {
+      if (Helper.BrowserConfiguration.IsFirefox())

```

Comment:
when will this get fixed? What are we doing with this issue when writing webtests?